lib/src/shim.dart

```diff
@@ -21,10 +22,12 @@ import 'utils/guid.dart';
 import 'utils/package_dir_detection.dart';
 import 'utils/per_file_pool.dart';
 
+final eol = Platform.isWindows ? '\r\n' : '\n';
```

Comment:
[nit] let's make this private.

Reply:
Doh; used to private-by-default! Fixed :)